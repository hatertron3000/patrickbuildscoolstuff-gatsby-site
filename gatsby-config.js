require("dotenv").config({
  path: `.env.${process.env.NODE_ENV}`,
})

console.log(`Env file will be: .env.${process.env.NODE_ENV}`)
/**
 * Configure your Gatsby site with this file.
 *
 * See: https://www.gatsbyjs.com/docs/gatsby-config/
 */

module.exports = {
  siteMetadata: {
    title: `Patrickbuildscoolstuff`
  },
  plugins: [
    'gatsby-plugin-react-helmet',
    {
      resolve: 'gatsby-source-contentful',
      options: {
        spaceId: process.env.CONTENTFUL_SPACE_ID,
        accessToken: process.env.CONTENTFUL_API_KEY,
      }
    },
    `gatsby-plugin-image`,
    `gatsby-plugin-sharp`,
    `gatsby-transformer-sharp`,
    {
      resolve: 'gatsby-plugin-s3',
      options: {
        bucketName: process.env.BUCKET_NAME,
      }
    }
  ],
}
