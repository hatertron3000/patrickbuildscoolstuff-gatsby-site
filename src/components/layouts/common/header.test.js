import React from "react"
import renderer from "react-test-renderer"

import Header from "./header"
import {siteMetadata} from "../../../../gatsby-config"


describe("Header", () => {
    it("matches snapshot", () => {
        const { title } = siteMetadata
        const tree = renderer
            .create(<Header title={title} />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})