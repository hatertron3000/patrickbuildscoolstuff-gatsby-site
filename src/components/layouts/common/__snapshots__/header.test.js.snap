// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<div>
  <div>
    <div>
      <a
        aria-current="page"
        className=""
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={Object {}}
      >
        Patrickbuildscoolstuff
      </a>
    </div>
  </div>
</div>
`;
